app/controllers/users_controller.rb

```diff
@@ -46,9 +53,10 @@ def create
 
   def update
     @user = scoped_users.find(params[:id])
+    @member = @user.as_member_of(current_organization)
     authorize @user
 
-    if @user.update(user_params)
+    if @user.update(user_params) && @member.update(tags: get_tags)
```

Comment:
this probably can be fixed/changed by the proposal in https://github.com/coopdevs/timeoverflow/pull/626/files#r623289771